king/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export CXX=${PREFIX}/bin/g++
+
+mkdir -p $PREFIX/bin
+cp ./KINGcode.tar.gz $PREFIX/bin/.
+wget http://people.virginia.edu/~wc9c/KING/KINGcode.tar.gz
+tar -xzvf KINGcode.tar.gz
+
+g++ KingCore.cpp
```

Comment:
I see different g++ compilation instructions on the [website](http://people.virginia.edu/~wc9c/KING/Download.htm). 

Try compiling this yourself first, and see if you can run the binary